scripts/ansible/roles/windows/openenclave/tasks/environment-setup-cross-arm.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
```

Comment:
The same small nit as above, the line:
```
---
```
is missing. I'd suggesting adding it for consistency between the Ansible `yml` files headers.